css/selectors/where.json

```
@@ -7,7 +7,14 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:where",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "83",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Experimental Web Platform Features",
```

For consistency, I'd suggest changing this. This hasn't been formalized yet, but see #6009 for discussion.

```suggestion
                  "name": "#enable-experimental-web-platform-features",
```